src/recording-graph.ts

```diff
@@ -49,12 +54,19 @@ interface GraphColors {
 
 export const getConfig = (
   { x: rawX, y: rawY, z: rawZ }: XYZData,
-  colors: GraphColors
+  colors: GraphColors,
+  lineStyles: GraphLineStyles,
+  graphLineWeight: GraphLineWeight
 ): ChartConfiguration<keyof ChartTypeRegistry, Pos[], string> => {
   const x = processDimensionData(rawX);
   const y = processDimensionData(rawY);
   const z = processDimensionData(rawZ);
-  const common = { borderWidth: 1, pointRadius: 0, pointHoverRadius: 0 };
+  const common = {
+    borderWidth:
+      graphLineWeight === "normal" ? 1.5 : graphLineWeight === "thin" ? 1 : 2.5,
```

Comment:
Consider use of fractional pixels.

Reply:
Fractional pixels no longer used as of [586de87](https://github.com/microbit-foundation/ml-trainer/pull/548/commits/586de871619c5c3b4e053d2e30402edd837429a7)